sh/install.sh

```diff
@@ -79,7 +113,16 @@ function install_python() {
         apt-get install python -y >> ${INSTALL_LOG} 2>&1
     elif [ $OPERATING_SYSTEM == "REDHAT" ]; then
         echo "Installing Python using yum"
-        yum install python -y >> ${INSTALL_LOG} 2>&1
+
+        file_name=/etc/redhat-release
+
+        # Check if file exists
+        if [ -f $file_name ]
```

Comment:
Is `/etc/redhat-release` not there on newer versions of redhat? If it is always expected on a redhat machine, maybe we should just log a error message on the else clause and not attempt to install python.

Reply:
I have updated the logic
/etc/redhat-release or /etc/system-release-cpe file will always be present